mono/metadata/marshal.c

```diff
@@ -8366,6 +8371,16 @@ mono_marshal_isinst_with_cache (MonoObject *obj, MonoClass *klass, uintptr_t *ca
 	return isinst;
 }
 
+static MonoObject*
+ves_icall_object_isinst (MonoObject *obj, MonoClass *klass)
```

Comment:
That's because we don't want to use `mono_object_isinst` in the runtime itself, and we don't want to introduce `MonoError` management in managed either.
I called it `ves_icall_...` because it makes it clearer that you can raise an exception as it is called by managed code, and not by native code.